default.lcf

```diff
@@ -113,3 +113,4 @@ log4j.category.javax.jmdns=OFF
 
 #log4j.category.jmri.jmrit.display.layoutEditor=DEBUG
 #log4j.category.jmri.jmrit.display.layoutEditor.LayoutBlock=DEBUG
+log4j.category.jmri.jmrix.sprog=DEBUG
```

Comment:
jmri.jmrix.sprog=DEBUG should not be checked in to JMRI/JMRI/master.

Reply:
has been reverted, thanks